src/Raven.Server/Documents/Handlers/DocumentHandler.cs

```diff
@@ -473,6 +483,8 @@ public async Task Delete()
 
                 var cmd = new DeleteDocumentCommand(id, changeVector, Database);
                 await Database.TxMerger.Enqueue(cmd);
+                if (TrafficWatchManager.HasRegisteredClients)
+                    AddStringToHttpContext(id, TrafficWatchChangeType.Documents);
```

Comment:
we can remove this since we log in the ctor now

Reply:
Done